docs/tutorials/google/calibration_faq.md

```diff
@@ -0,0 +1,311 @@
+by four. This option might be useful to quickly check for the state of a device or benchmark particular qubit configurations before 
+doing the actual experiment.
+
+For example, if you have a circuit on a line of 5 qubits, there are 4 unique two-qubit gates and 7 unique moments with different 
+combinations of two-qubit gates. If you use `cirq_google.prepare_floquet_characterization_for_operations`, this will result
+in 2 characterizations (since 2 unique gates can be put into a single moment) . If you use `cirq_google.prepare_characterization_for_moments`
+with `merge_subsets=False` this will result in 7 characterizations.
+
+## What if the compensation based on single-qubit phases is not enough?
+
+The provided compilation `cirq_google.make_zeta_chi_gamma_compensation_for_moments` uses Z gates to compensate for errors. 
+For the `cirq.SQRT_ISWAP` gate this does not affect duration of a circuit because the added Z gates are merged with the 
+already-existing Z gate pulses after submitting circuits to server. This compilation method that uses just Z gates can only 
+compensate for 3 angles: ζ, χ and γ.
+
+Compensation for the remaining two parameters θ and φ can’t be realized in a simple way.  Some experiments can be modified 
+to adjust to these errors by modifying circuit construction, compilation, or problem statement, but this must be handled by the
+user and cannot be done automatically.
+
+For these advanced use cases, the `cirq_google.run_calibration` method can be called directly with a list of 
+`cirq_google.PhasedFSimCalibrationRequest` objects.  This will allow you to call the Calibration API directly to specify 
+customized layers for characterizations.
+
+```python
+@dataclasses.dataclass(frozen=True)
+class PhasedFSimCalibrationRequest(abc.ABC):
+    pairs: Tuple[Tuple[Qid, Qid], ...]   # a list of pairs of in a single moment to characterize
+    gate: Gate                                    # desired gate (must be cirq.ISWAP ** -0.5 for Floquet)
+    options: PhasedFSimCalibrationOptions  # detailed adjustments
+               # either FloquetPhasedFSimCalibrationOptions or LocalXEBPhasedFSimCalibrationOptions
+```
+
+After triggering this calibration with the `cirq_google.run_calibrations` method, the `PhasedFSimCalibrationResult` is returned
+with a parameters field that contains the unitary parameters of all the requested gates. The object also contains additional
+execution metadata which might be useful.
+
+## Floquet calibration fails with a message: Readout errors above the tolerance for qubit pairs (...), what can be done?
+
+This is an automated check performed before the actual Floquet calibration on QCS used to detect if a particular gate did not 
+drift too much outside of reasonable error levels. When the check fails, the following exception is reported:
+
+```python
+PhasedFSimCalibrationError: Readout errors above the tolerance for qubit pairs ((8, 3), (8, 4))
+```
```

Comment:
Is everything alright with Markdown here, looks like github lost context?

Reply:
I think so... if it displays poorly on the site I can update.